### PR TITLE
Add Android TV 0.17 blog post

### PR DESCRIPTION
**Pull request by nielsvanvelzen**
Blog post for the upcoming 0.17 release of jf-atv.

> [!CAUTION]
> **!!Do not merge!!** - I'll merge it myself when the release is ready.

### CODE REVIEW

**Comment by nielsvanvelzen** [comment]
image

File: blog/2024/08-03-android-tv-17/index.mdx [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+
+![Screenshot of item details in the Android TV app](./item-details.png)
+
+### Playback
+
+The current video player was improved by adding support for more audio codecs, including WAV, Vorbis and PCM. Additionally, timeouts for
+player errors were increased to give transcode and remux tasks more time to start. This improves playback for media stored on network
+drives and BDMV files that require unpacking.
+
+Audio normalization support was added to automatically adjust volume levels. This feature prevents sudden volume changes between songs. To
```
Comment (anthonylavado) [inline blog/2024/08-03-android-tv-17/index.mdx:34]:
```suggestion
Audio normalization support was added to automatically adjust volume levels for music playback. This feature prevents sudden volume changes between songs. To
```

**Review by anthonylavado — CHANGES_REQUESTED** [review]
Otherwise it looks good. Can't wait!

File: blog/2024/08-03-android-tv-17/index.mdx [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+The screensaver now includes a feature to filter items based on their age rating. By default, only items with an age rating up to 13 will
+be displayed, ensuring a family-friendly environment. Age rating options can be changed in the preferences under the customization tab.
+
+Additionally, the screensaver has been enhanced to display a logo of the movie or series being shown.
+
+![Screenshot of the updated screensaver design in the Android TV app](./screensaver.png)
+
+### Other changes
+
+Series and seasons now display their respective specials, which were previously available only for movies. Additionally, the search
```
Comment (Shadowghost) [inline blog/2024/08-03-android-tv-17/index.mdx:48]: Are we talking Specials or Extras here?

**Review by Shadowghost — APPROVED** [review]

File: blog/2024/08-03-android-tv-17/index.mdx (hunk 2 above)
Comment (nielsvanvelzen) [reply blog/2024/08-03-android-tv-17/index.mdx:48]: What web calls "Special Features"

**Pull request merged**